packages/samples/mvp-dapp/src/exception.ts

```diff
@@ -0,0 +1,29 @@
+import { Context, HttpError, Next } from 'koa'
+import { MvpResponse } from './response'
+
+export function handleException() {
+  return async (ctx: Context, next: Next) => {
+    try {
+      await next()
+    } catch (err) {
+      if (err instanceof HttpError) {
+        ctx.body = { message: err.message }
+      } else if (err instanceof MvpError) {
+        ctx.status = 200
+        ctx.body = MvpResponse.err(err.message, err.code)
+      } else {
+        ctx.status = 500
+        ctx.body = { message: 'Internal server error' }
```

Comment:
Shall we need to print the error or return the detailed error when in the development stage?
It may easy to locate some unexpected exceptions.

Reply:
It's a MVP DApp, we may leave it simple